### ISSUE TITLE
Touchscreen controls on Android

### ISSUE THREAD

**Issue by ApocalypsjeNL** [OpenRCT2/OpenRCT2#6146]
**OS:** Android 7.1.2
**Version:** 0.1.1
**Commit/Build:** 591aa8a

Is it possible to create touchscreen controls like on RCT Classic? It's a pain in the ass to play it now on android without having to plug a mouse into a otg cable.

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Multiplayer?

**Comment by PFCKrutonium** [comment]
Yes. Soon? If someone decides to make it happen, Yes. Has sombody? Not that I am aware of. Having an issue on the tracker will make it happen sooner though! 👍 (Not sure if duplicate)

**Comment by kripton** [comment]
I don't know how much code OpenRCT2 and OpenTTD share, but OpenTTD has a pretty playable Android "port": https://github.com/pelya/openttd-android

**Comment by GeorgeGamez** [comment]
There is [a plugin](https://openrct2plugins.org/plugin/R_kgDOHKMz3A/openrct2-touch-plugin) that can make it slightly more usable, but we still need a major improvement on touch controls.
